src/libs/actions/PersonalDetails.js

```diff
@@ -71,7 +71,7 @@ function getDisplayName(login, personalDetail) {
  * @returns {String}
  */
 function getMaxCharacterError(isError) {
```

Comment:
We use `getMaxCharacterError` in `validate` function and add error to the errors object returned by `validate`.

Additionally, the `getMaxCharacterError` function seems to be used in `RequestCallPage.js`, which currently isn't using `Form`.

Please advise?

Reply:
@liyamahendra we could do something like this instead

```
if (hasFirstNameError) {
    errors.firstName = Localize.translateLocal('personalDetails.error.characterLimit', {limit: CONST.PROFILE_INPUTS_CHARACTER_LIMIT});
}
```

Reply:
@rushatgabhane we need the `IF` check otherwise the error object will have that property (firstName, lastName and so on) added inside it and the form will show error even when there is no error.